PubHubs/Issues/pubhubsaccount/description.xml

```diff
@@ -0,0 +1,55 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>PubHubs Account</en>
+		<nl>PubHubs Account</nl>
+	</Name>
+	<ShortName>
+		<en>PubHubs Account</en>
+		<nl>PubHubs Account</nl>
+	</ShortName>
+	<SchemeManager>pbdf</SchemeManager>
+	<IssuerID>PubHubs</IssuerID>
+	<CredentialID>pubhubsaccount</CredentialID>
+	<Description>
+		<en>Your PubHubs account</en>
+		<nl>Je PubHubs account</nl>
+	</Description>
+	<ShouldBeSingleton>false</ShouldBeSingleton>
+	<IssueURL>
+		<en>https://privacybydesign.foundation/attribute-index/en/pbdf.PubHubs.pubhubsaccount.html</en>
+		<nl>https://privacybydesign.foundation/attribute-index/nl/pbdf.PubHubs.pubhubsaccount.html</nl>
```

Comment:
The IRMA app uses these URLs to send the user to when she wants to refresh her credential, or load it during a disclosure session when she doesn't yet have the credential. As such, these two URLs won't work: in case of `pbdf` credentials such as this one, the attribute index doesn't offer the possibility to issue the credential ([example](https://privacybydesign.foundation/attribute-index/en/pbdf.sidn-pbdf.email.html)). It only does that for `irma-demo` credentials (since for those the private keys are known).

Is there a publically reachable URL of a PubHub website where users can go to start the issuance flow of these credentials? If so, those URLs should be here. If not, there should be no `IssueURL` at all.

Reply:
Aaah, good point, thanks for the explanation. Makes sense, so I've removed the url